webextensions/manifest/background.json

```diff
@@ -64,6 +64,7 @@
               "firefox": {
                 "version_added": "48",
                 "notes": [
+                  "From Firefox 109, persistent and non-persistent pages are fully supported for Manifest V2 and V3.",
```

Comment:
MV3 only supports non-persistent background pages. It does not support the `persistent` key. Drop this line.

Also, as a cleanup, drop the two notes below about the preferences. Now that the feature has officially shipped, the prefs in older browser versions are not relevant.